rosdep/python.yaml

```diff
@@ -5786,6 +5786,10 @@ python3-psutil:
   rhel: ['python%{python3_pkgversion}-psutil']
   slackware: [psutil]
   ubuntu: [python3-psutil]
+python3-pyassimp:
+  debian: [python3-pyassimp]
+  fedora: [assimp-python3]
+  ubuntu: [python3-pyassimp]
```

Comment:
It looks like python3-assimp is only available on Ubuntu since bionic.  Since we still support Xenial for Kinetic, I think we need to change this to:

```
  ubuntu:
    '*': [python3-pyassimp]
    xenial: null
```